nodes/alarm_server.py

```diff
@@ -224,31 +120,47 @@ def _create_alarm_handlers(self):
             alarm_name = handler.alarm_name
             severity = handler.severity_required
 
-            if alarm_name not in self.alarms:
-                self.alarms[alarm_name] = Alarm.blank(alarm_name)
-            self.alarms[alarm_name].raised = h.initally_raised
+            # Set initial state if necessary (could have already been added while creating metas)
+            if hasattr(h, 'initial_alarm'):
```

Comment:
If you look later on there is a call to the alarm's add_callback method. That method does call all of the currently added callbacks. This makes all of the alarm handler callbacks process them at startup. You're right about it not being published to /alarm/updates. 

Since the initial alarm one of the first things to come up, I it should be the responsibility of alarm listeners to assume that the initial alarm will be there before they are launched and check for it manually (service call) if they need to. Any reasonable objections?